### PR TITLE
Add sklearn-crfsuite

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sklearn-crfsuite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```
Comment (synapticarbors) [inline recipes/sklearn-crfsuite/meta.yaml:21]: The `setup.py` specifies version restrictions that should be replicated here:

https://github.com/TeamHG-Memex/sklearn-crfsuite/blob/master/setup.py#L15-L19

File: recipes/sklearn-crfsuite/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2010-2020 Google, Inc. http://angularjs.org
```
Comment (synapticarbors) [inline recipes/sklearn-crfsuite/LICENSE:3]: Is this license correct for this package? I don't see an actual copy of the license upstream in the source repo. Please also submit an issue in the github repo asking them to actually include a license in their source distribution and include a link in the `meta.yaml` to the issue as a reminder for future updates to the feedstock.

**Comment by ndmaxar** [comment]
@oblute , @rluria14 agree to be maintainers?

File: recipes/sklearn-crfsuite/meta.yaml (hunk 1 above)
Comment (ndmaxar) [reply recipes/sklearn-crfsuite/meta.yaml:21]: I've added the pins.

File: recipes/sklearn-crfsuite/LICENSE (hunk 2 above)
Comment (ndmaxar) [reply recipes/sklearn-crfsuite/LICENSE]: Removed license file, and have created an issue in the repo. [issue](https://github.com/TeamHG-Memex/sklearn-crfsuite/issues/48)

**Pull request merged**